ablator/main/proto.py

```diff
@@ -32,14 +54,29 @@ def pre_train_setup(self):
 
     def _init_state(self):
         """
-        initialize the data state of the wrapper to force downloading and processing any data artifacts
+        Initialize the data state of the wrapper to force downloading and processing any data artifacts
         in the main train process as opposed to inside the wrapper.
         """
         self.pre_train_setup()
         mock_wrapper = copy.deepcopy(self.wrapper)
         mock_wrapper._init_state(run_config=copy.deepcopy(self.run_config), debug=True)
 
     def launch(self, debug: bool = False):
+        """
+        Initialize the data state of the wrapper and train the model inside the wrapper, then sync training
```

Comment:
should we add comments for sync here? since sync does nothing in proto trainer

Reply:
I think it will be added after

Reply:
It's just a prototype class so I think it's not important whether to mention sync or not.